Sources/PostgresNIO/Connection/PostgresConnection+CopyFrom.swift

```diff
@@ -0,0 +1,234 @@
+/// Handle to send data for a `COPY ... FROM STDIN` query to the backend.
+public struct PostgresCopyFromWriter: Sendable {
+    /// The backend failed the copy data transfer, which means that no more data sent by the frontend would be processed.
+    ///
+    /// The `PostgresCopyFromWriter` should cancel the data transfer.
+    public struct CopyCancellationError: Error {
```

Comment:
Where is the benefit of wrapping this here? Shouldn't we use the PSQLError directly?